### ISSUE TITLE
NTR: Thymidine phosphorylase enzyme activity low in leukocytes (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8138]
New term request Thymidine phosphorylase enzyme activity low in leukocytes (segal_050428154107): <10%  UMLS: 1864225

**Comment by pnrobinson** [comment]
@MickeySegal
this is not leukocyte specific, see, e.g., https://pubmed.ncbi.nlm.nih.gov/18787099/

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000570
- label: Reduced tissue thymidine phosphorylase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:10852545
- def: Activity of thymidine phosphorylase (EC 2.4.2.4) in the tissues below the lower limit of normal. The activity can be measured in multiple tissues including leukocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8138
- synonyms: Reduced leukocyte thymidine phosphorylase activity(NONE)
